src/analytics/api/views/data.py

```
@@ -442,9 +299,8 @@ def get(self):
                 AirQoRequests.Status.HTTP_200_OK,
             )
 
-        except Exception as ex:
-            print(ex)
-            traceback.print_exc()
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)